.circleci/config.yml

```diff
@@ -49,22 +49,22 @@ jobs:
 
     - restore_cache:
         keys:
-        - v1-librdkafka-v0.11.5
+        - v1-librdkafka-v1.0.0-RC4
```

Comment:
This has been updated because CI was failing. The latest version of our Kafka integration has this version as a requirement.